app/templates/home.html

```diff
@@ -4,8 +4,10 @@
 
 {% block main_column %}
 
-  {% if error %}
-    <p>{{ error }}</p>
+  {% if not current_user.is_superuser %}
```

Comment:
Not sure I'm keen on this approach. It works but it's a bit strange that the UI determine if the user is a super user, and also we set this flag to show a permission error message.

I'm thinking, maybe we can set something like `req.errors` or something like that.

Reply:
This is just a stop-gap solution while we wait for proper authorization in the API. Then we'll get the permissions from the API and check those instead.

Reply:
👍